packages/compartment-mapper/package.json

```diff
@@ -14,7 +14,9 @@
   "scripts": {
     "build": "rollup --config rollup.config.js",
     "clean": "rm -rf dist",
-    "lint": "eslint '**/*.js'",
+    "lint": "yarn lint:types && yarn lint-check",
+    "lint:types": "tsc -p jsconfig.json",
```

Comment:
We’ll likely want to eventually move this to `build` and generate `index.d.ts` for consumption across package boundaries. But that can be deferred. We would need a cross-package TypeScript integration test to validate it.

Reply:
On that topic I got as far as looking for precedent; the only one I found was in `@agoric/base64`:

```json
    "build:types": "tsc src/*.js --declaration --allowJs --emitDeclarationOnly --outDir types",
```
https://github.com/Agoric/agoric-sdk/blob/master/packages/base64/package.json#L24

That didn't seem like a very strong precedent, so I left it alone.

Reply:
Yes, that’s new. @michealfig, Do you think we should do this in general for “proper” packages?

Reply:
I haven't had success with generating types yet, but if you've found something that works, go ahead.